packages/tasks/src/hardware.ts

```diff
@@ -0,0 +1,309 @@
+			},
+			"RTX 6000 Ada": {
+				tflops: 91.1,
+				memory: [48],
+			},
+			A100: {
+				tflops: 77.97,
+				memory: [80, 40],
+			},
+			A40: {
+				tflops: 37.42,
+				memory: [48],
+			},
+			A10: {
+				tflops: 31.24,
+				memory: [24],
+			},
+			T4: {
+				tflops: 65.13,
+				memory: [16],
+			},
+			"RTX 4090": {
+				tflops: 82.58,
+				memory: [24],
+			},
+			"RTX 4080 SUPER": {
+				tflops: 52.2,
+				memory: [16],
+			},
+			"RTX 4080": {
+				tflops: 48.7,
+				memory: [16],
+			},
+			"RTX 4070": {
+				tflops: 29.15,
+				memory: [12],
+			},
+			"RTX 4070 Ti": {
+				tflops: 40.09,
+				memory: [12],
+			},
+			"RTX 4070 Super": {
+				tflops: 35.48,
+				memory: [12],
+			},
+			"RTX 4070 Ti Super": {
+				tflops: 44.1,
+				memory: [16],
+			},
+			"RTX 3090": {
+				tflops: 35.58,
+				memory: [24],
+			},
+			"RTX 3090 Ti": {
+				tflops: 40,
+				memory: [24],
+			},
+			"RTX 3080 Ti": {
+				tflops: 34.1,
+				memory: [12],
+			},
+			"RTX 3080": {
+				tflops: 30.6,
+				memory: [12, 10],
+			},
+		},
+		AMD: {
+			MI300: {
+				tflops: 383.0,
+				memory: [192],
+			},
+			MI250: {
+				tflops: 362.1,
+				memory: [128],
+			},
+			MI210: {
+				tflops: 181.0,
+				memory: [64],
+			},
+			"RX 7900 XTX": {
+				tflops: 122.8,
+				memory: [24],
+			},
+			"RX 7900 XT": {
+				tflops: 103.0,
+				memory: [20],
+			},
+			"RX 7900 GRE": {
+				tflops: 91.96,
+				memory: [16],
+			},
+			"RX 7800 XT": {
+				tflops: 74.65,
+				memory: [16],
+			},
+			"RX 7700 XT": {
+				tflops: 70.34,
+				memory: [12],
+			},
+			"RX 7600 XT": {
+				tflops: 45.14,
+				memory: [16, 8],
+			},
+		},
+	},
+	CPU: {
+		Intel: {
+			"Xeon 4th Generation (Sapphire Rapids)": {
+				tflops: 3.8,
+			},
+			"Xeon 3th Generation (Ice Lake)": {
+				tflops: 3.6,
+			},
+			"Xeon 2th Generation (Cascade Lake)": {
+				tflops: 3,
+			},
+			"Intel Core 13th Generation (i9)": {
+				tflops: 3.8,
+			},
+			"Intel Core 13th Generation (i7)": {
+				tflops: 3.6,
+			},
+			"Intel Core 13th Generation (i5)": {
+				tflops: 3,
+			},
+			"Intel Core 13th Generation (i3)": {
+				tflops: 2,
+			},
+			"Intel Core 12th Generation (i9)": {
+				tflops: 2,
+			},
+			"Intel Core 12th Generation (i7)": {
+				tflops: 2,
+			},
+			"Intel Core 12th Generation (i5)": {
+				tflops: 2,
+			},
+			"Intel Core 12th Generation (i3)": {
+				tflops: 1,
+			},
+			"Intel Core 11th Generation (i9)": {
+				tflops: 1,
+			},
+			"Intel Core 11th Generation (i7)": {
+				tflops: 0.76,
+			},
+			"Intel Core 11th Generation (i5)": {
+				tflops: 0.76,
+			},
+			"Intel Core 11th Generation (i3)": {
+				tflops: 0.76,
+			},
+			"Intel Core 10th Generation (i9)": {
+				tflops: 0.76,
+			},
+			"Intel Core 10th Generation (i7)": {
+				tflops: 0.76,
+			},
+			"Intel Core 10th Generation (i5)": {
+				tflops: 0.76,
+			},
+			"Intel Core 10th Generation (i3)": {
+				tflops: 0.76,
+			},
+		},
+		AMD: {
+			"EPYC 4th Generation (Genoa)": {
+				tflops: 4,
```

Comment:
```suggestion
				tflops: 5.3,
```
not clear, source: https://www.hpcwire.com/2022/11/10/amds-4th-gen-epyc-genoa-96-5nm-cores-across-12-compute-chiplets/

Reply:
Here it gives 4.6 https://gadgetversus.com/processor/amd-epyc-9654-specs/